modules/lib/lib-schema/src/main/resources/lib/xp/examples/schema/updateContentType.js

```diff
@@ -0,0 +1,75 @@
+var schemaLib = require('/lib/xp/schema');
+var assert = require('/lib/xp/testing');
+
+/* global log*/
+
+let resource = `<?xml version="1.0" encoding="UTF-8"?>
+        <content-type xmlns="urn:enonic:xp:model:1.0">
+            <display-name>My Tag</display-name>
+            <description>My description</description>
+            <super-type>base:structured</super-type>
+            <is-abstract>false</is-abstract>
+            <is-final>true</is-final>
+            <allow-child-content>true</allow-child-content>
+            
+            <x-data name="myapp:address"/>
+            
+            <form>
+                <input name="tag_unlimited" type="Tag">
+                    <label>Tag, unlimited occurrences</label>
+                    <immutable>false</immutable>
+                    <indexed>true</indexed>
+                    <custom-text>Custom text</custom-text>
+                    <occurrences minimum="0" maximum="0"/>
+                    <help-text>Some help text</help-text>
+                </input>
+            </form>
+        </content-type>`
+
+// BEGIN
+// Update virtual content type.
+var result = schemaLib.updateSchema({
+    name: 'myapp:mytype',
+    type: 'CONTENT_TYPE',
+    resource: resource
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9736743)
```suggestion
    resource
```